commands/create_cluster.go

```diff
@@ -222,6 +237,12 @@ func createClusterExecutionOutput(cmd *cobra.Command, args []string) {
 			headline = "The cluster could not be created."
 			subtext = "You might try again in a few moments. If that doesn't work, please contact the Giant Swarm support team."
 			subtext += " Sorry for the inconvenience!"
+
+			if richErrorOK {
+				subtext += "\n\nDetails:\n"
+				subtext += richError.Message()
+			}
```

Comment:
Isn't it better to add `else` and print any error even if the output is ugly? How user can create an issue is something isn't working?

Reply:
You mean, even if the error was not of type `errgo.Err`, print these details?

Reply:
Yes. At least `error.Error()`.